packages/www/pages/_app.tsx

```diff
@@ -65,12 +66,8 @@ const livepeerClient = createReactClient({
     // we intentionally provide no API key so any requests requiring auth will fail
     // eventually should move to using JWT from user's login
     apiKey: "",
-    baseUrl: isStaging()
-      ? "https://livepeer.monster/api"
-      : "https://livepeer.studio/api",
-    webrtcIngestBaseUrl: isStaging()
-      ? "https://webrtc.livepeer.monster/webrtc"
-      : "https://webrtc.livepeer.studio/webrtc",
+    baseUrl: `${getEndpoint()}/api`,
+    webrtcIngestBaseUrl: `${getEndpoint()}/webrtc`,
```

Comment:
Neat! Keep in mind that the webrtc endpoints had a `webrtc.` subdomain though. maybe specify it as an optional arg to `getEndpoint()`? Or does it work without it?

Reply:
I'm still confused what webrtc. is for and was planning to let this get to staging and verify that hitting the same URLs via `playback.livepeer.monster` resulted in the same result :shrug:

Reply:
Yeah, it's the same, and we didn't have a giant reason for doing playback different from ingest. The only reason was for possible separation of concerns, if we ever want to route ingest differently somehow. But this looks good

Reply:
Right on. I like using the same one as much as possible, for exactly the use case I'm working on of making it easy to set up a single-node server that does everything.